.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~~not required~~, ~~covered~~ . Link --> #XYZ
+- [ ] **dev PR** provided, ~~not required~~ Link --> #XYZ
```

Comment:
Question (does not have to result in a change in this PR, I just want to make sure I understand): for the development PR, will we add a link to self here?

Reply:
Regardless, I expect the tooling to work no matter where the link is on the line, and no matter what the source syntax is (`#nnnn` or a bare link or a Markdown link).

Reply:
@mpg Ideally that should only be needed in backport-prs. So for a development PR if we include the comments that link could be

~~~~~~
- [ x ] **dev PR** ~~provided~~, not required. This is the dev PR.  Link --> #XYZ
~~~~~~~

But as @gilles-peskine-arm the tooling should work even if someone does that

~~~~~~
- [ x ] **dev PR** provided, ~~not required~~.  Link --> #SELF_NO
~~~~~~~